### DISCUSSION TITLE
a plausible import/export format

### DISCUSSION THREAD

**Discussion by jesperordrup** [plausible/analytics#1949] [Anything Else]
Hi Plausible

Have you considered a plausible import/export format? This would allow anyone to copy or move between installations or backup/restore stats. Also it could be used for anyone as a standard way to rearrange data exported from other systems. Just make a plausible import format and then it works :-) 

This will allow for moving stuff - even without developing. 

thx
Jesper

**Comment by metmarkosaric** [comment]
thanks for the feedback @jesperordrup!

we'd like to expand the export/import to include self-hosted and other tools too. there's a thread for all the import / export topics here: https://github.com/plausible/analytics/discussions/211. until now, we've managed to build an export tool for both the cloud and self-hosted which was the most popular request. we're recently introduced GA import tool on the cloud and will work on introducing GA import tool in the next version of self-hosted as this is another popular request.

next up on the export / import topic in terms of popularity of requests are are a more general import tool that supports other analytics tools, GA4 import and a tool that allows data transfers between self-hosted and cloud. 

we're a small team with limited resources and this is a major undertaking so things will take time. i cannot make any promise on when we'll be able to work on this as there are some other more frequently requested features we're working on at the moment. we'll post any updates on this topic in the thread linked above. thanks for understanding!

**Reply by jesperordrup** [reply]
Thanks @metmarkosaric - I fully understand. Looking forward to updates
